tests/modules/individuals/test_models.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+# pylint: disable=invalid-name,missing-docstring
+
+
+def test_individual_add_remove_encounters(encounter_1, encounter_2, empty_individual):
+
+    assert len(empty_individual.encounters) == 0
+
+    empty_individual.add_encounter(encounter_1)
+    assert len(empty_individual.get_encounters()) == 1
+
+    empty_individual.add_encounter(encounter_2)
+    assert len(empty_individual.get_encounters()) == 2
+
+    assert encounter_1.individual is empty_individual
+    assert encounter_2.individual is empty_individual
+
+    empty_individual.remove_encounter(encounter_2)
+    assert len(empty_individual.get_encounters()) == 1
+
+    empty_individual.remove_encounter(encounter_1)
+
+    encounter_list = [encounter_1, encounter_2]
+    empty_individual.add_encounters(encounter_list)
+    assert len(empty_individual.get_encounters()) == 2
+    assert encounter_1.individual is empty_individual
+    assert encounter_2.individual is empty_individual
+
+
+def test_individual_ownership(encounter_1, encounter_2, empty_individual):
+    empty_individual.encounters.append(encounter_1)
```

Comment:
likewise, I may be wrong but I think that if you run just the individual pytest a few times, this test will be run before the one above and when it asserts that the len of get_encounters() is 0 that should fail

Reply:
Also fixed to restore test objects to original state.